DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1595,14 +1595,14 @@ private void MapSharer_HandleMapDownloadComplete(SHA1EventArgs e)
         {
             string mapFileName = MapSharer.GetMapFileName(e.SHA1, e.MapName);
             Logger.Log("Map " + mapFileName + " downloaded, parsing.");
-            string mapPath = "Maps/Custom/" + mapFileName;
+            string mapPath = SafePath.CombineFilePath(MapLoader.CustomMapsDirectory, mapFileName);
```

Comment:
I think this might break things since the string might be used for other stuff that relies on the previous exact value e.g. section headers in ini files. I recommend only using SafePath when evaluating the path (when actually accessing the filesystem).